sudo/lib/exec/monitor.rs

```diff
@@ -103,7 +118,9 @@ impl<'a> MonitorClosure<'a> {
 
         // Put the command in its own process group.
         let command_pgrp = command_pid;
-        setpgid(command_pid, command_pgrp).ok();
+        if let Err(_err) = setpgid(command_pid, command_pgrp) {
```

Comment:
does this need the underscore prefix because you get a warning when the dev feature is disabled?

Reply:
@japaric do you think there's a way to avoid the warning and the underscore that effectively lets this be a noop if dev is disabled?

Reply:
normally I would write
``` rust
if cfg!(feature = "dev") {
    // do stuff with macro args
}
```

instead of 
``` rust
#[cfg(feature = "dev")]
{
    // do stuff with macro args
}
```
which has the problem you are observing.

if that's not possible, I guess the other option is to put a bunch of `drop` in the other branch of the conditional :sweat_smile: 
``` rust
#[cfg(not(feature = "dev"))]
{
   $(drop(&$arg);)*
}
```